projects/angular/src/layout/breadcrumbs/breadcrumbs.html

```diff
@@ -0,0 +1,17 @@
+<div role="list" class="clr-breadcrumb-menu">
+  <ng-container *ngIf="items?.length">
+    <clr-breadcrumb-item *ngIf="items.length > max && !isExpanded">
+      <span (click)="expand()" (keydown)="expand()" class="clr-breadcrumb-expand">
+        <cds-icon shape="ellipsis-horizontal"></cds-icon>
+      </span>
+    </clr-breadcrumb-item>
+    <ng-container *ngFor="let breadcrumb of items | slice: -limit : items.length; let isLastItem=last">
+      <clr-breadcrumb-item>
+        <a *ngIf="!isLastItem" (click)="handleItemClick(breadcrumb)" [routerLink]="breadcrumb?.url">
```

Comment:
I think binding a `url` to `[routerLink]` can be confusing. It could also likely be an external url depending on how VCF components are integrated.

This component should support both `[routerLink]` and `[href]`.

Reply:
Added support for `[href]`, `target` and `queryParams`